### PR TITLE
unixPB: Install rng-tools to fix low entropy

### PR DESCRIPTION
**Pull request by aswinkr77**
Cryptography tests are failing as not enough random information is being generated on older distributions, causing tests to timeout.
So installing and enabling rng-tools and its services fix the issues. 

##### Fixes:
 - https://github.com/adoptium/aqa-tests/pull/4474
 - https://github.com/eclipse-openj9/openj9/issues/16720

##### Checklist

- [ ] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Review by karianna — APPROVED** [review]

**Comment by AdamBrousseau** [comment]
Do we need another review on this or just a merge?
cc @Haroon-Khel

**Comment by sxa** [comment]
Needs a second review first :-)

A couple of questions from me:
- Will these rules work ok in a docker container? or is it only applicable on the host of such a container?
- This won't be run on other distributions such as Debian/RHEL as-is - is that intentional?

**Comment by aswinkr77** [comment]
> Needs a second review first :-)
> 
> A couple of questions from me:
> 
> * Will these rules work ok in a docker container? or is it only applicable on the host of such a container?
> * This won't be run on other distributions such as Debian/RHEL as-is - is that intentional?

- We've never tested this on a container cause we don't run these tests on containers.
- The PB will run on Ubuntu, Rhel, Cent and SLES, not Debian. We've never come across the entropy issue on a Debian machine, mostly it happened with the other four distros.

**Comment by sxa** [comment]
I'm on vacation until Tuesday but w will need to verify that these changes don't cause as problem in containers. Our build are performed in containers built with the playbooks so it's critical that this won't fail to execute or break anything there.
I would hope that it works be ok, but I'm nervous enough to want to be be verified first 😉
Surprised it hasn't been deemed necessary on Debian though - I wonder if they're doing anything different 🤷

**Review by karianna — APPROVED** [review]

**Comment by AdamBrousseau** [comment]
@sxa does one of those automatic checks test the docker compile? If not will Adopt have the capacity to test what you want to verify? I don't think we have any Debian machines so it is untested on our end and unknown if needed.

**Comment by sxa** [comment]
> I don't think we have any Debian machines so it is untested on our end and unknown if needed.

Understood. Thanks for clarifying

> @sxa does one of those automatic checks test the docker compile? 

No but we have automated triggers that try to run a docker build from the updated playbooks when things are merged. Normally it's not something that would cause a problem but when we're starting services and doing stuff related to things in `/dev` it makes me nervous that a docker environment is a bit special in those respects. If you're able to run a `docker build` with at least one of the dockerfiles in https://github.com/adoptium/infrastructure/tree/master/ansible/docker using these changes (Probably a CentOS one as a preference) that would be good.

**Comment by aswinkr77** [comment]
@sxa ran the PB on the centos6 image. Build openjdk11. Everything went fine.

**Review by sxa — APPROVED** [review]
> @sxa ran the PB on the centos6 image. Build openjdk11. Everything went fine.

OK That basis I'm ok with giving this a shot :-) My only comment would be that it might be worth putting the package installs into the common role where we already have a good abstraction for the distributions.

I'm in two minds about whether to request that we have the `adoptopenjdk` tag on this one since it could make a notable change to end users systems. If anyone else has a view on these feel free to jump in :-)

@andrew-m-leonard FYI - I don't /think/ this should affect build reproducibility but let me know if you want to perform any additional checks prior to merging.

**Comment by andrew-m-leonard** [comment]
Sounds reasonable to me

**Comment by sxa** [comment]
I'll plan to merge this tomorrow after the overnight build cycles in the absence of any objections.

**Pull request merged**
